src/components/AccountAssets/CustomTrustline.tsx

```diff
@@ -51,6 +51,10 @@ function CustomTrustlineDialog(props: Props) {
           onChange={event => setIssuerPublicKey(event.target.value)}
         />
         <TextField
+          inputProps={{
+            pattern: "[0-9]*",
```

Comment:
Using `pattern` seems dangerous. We need to allow entering decimal places, but depending on the user's locale we might run into trouble (in non-English locales a decimal point could also be a comma or a single quote, for instance).

Is there a good reason to use the pattern?

Reply:
The pattern property is necessary to make it look like [this](https://user-images.githubusercontent.com/27284008/65964136-8ae68c80-e45c-11e9-9412-92ad1e3a4b0b.png).
Without it, the iOS keyboard won't show only the numbers but other characters as well, if I remember correctly.

Reply:
True, it does show the decimal point. Just tested it!
Have you tried it on Android, too?

Reply:
Yes I did. And I was pleasantly surprised because it has a similar effect on the Keyboard (showing a huge block with numbers) instead of showing a number row + normal letters.